NewForce-at-Mountwest/bangazon-site-the-blue-ferrets
# Users can view and edit their account settings

# Description

A user can now view the appropriate fields in account settings. A user can edit the fields listed on the ticket and update the database accordingly.

To Test:
Register a new user.
Go to my account in the nav bar.
You should see the information you registered with prepopulated in the form.
Edit the Phone number field and click the save affordance to ensure it posts correctly. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

To Test:
Register a new user.
Go to my account in the nav bar.
You should see the information you registered with prepopulated in the form.
Edit the Phone number field and click the save affordance to ensure it posts correctly.



# Checklist:
